apps/website/src/content/docs/informationpanel.mdx

```diff
@@ -17,6 +14,31 @@ The information panel is a pattern used primarily in tables. It allows viewing o
 
 The info panel is reserved for complex tables, because it allows the user to gradually explore the content without overwhelming them with too many details. It also helps avoiding the unpleasant horizontal scrolling when the table gets too wide. Not every table requires an information panel.
 
+## Usage
+
+### Composition API
+
+The Information Panel component is made up of customizable subcomponents. Here’s what the basic usage looks like:
+
+```tsx
+<InformationPanelWrapper>
+  <Table ... /> // component to overlay
+  <InformationPanel isOpen={true}>
+    <InformationPanelHeader onClose={() => {}}>
+      <Text variant='subheading'>InfoPanel heading</Text>
+    </InformationPanelHeader>
+      <InformationPanelBody>
+      <Text>Metadata</Text>
+      <InformationPanelContent ... />
+      <Text>Attributes</Text>
+      <InformationPanelContent ... />
+    </InformationPanelBody>
+ </InformationPanel>
+</InformationPanelWrapper>
+```
+
+By default, the Information Panel component is closed. It can be opened by setting `isOpen={true}`. The orientation and size of the panel can be set using the `orientation` and `resizable` prop respectively.
```

Comment:
Some code demos from the stories can be brought here. You can either make each story as a new example, or combine multiple stories in an example if you feel a separate example is unnecessary.

i.e. instead of just mentioning the `orientation` and `resizable` props, it would be good to have some small demos.